### ISSUE TITLE
Move court case decisions from alphabetical list to dedicated pages

### ISSUE THREAD

**Issue by zsmith-fec** [fecgov/fec-cms#4246]
### Summary

**What we're after:**
We need to move the court case decisions from the [alphabetical list](https://www.fec.gov/legal-resources/court-cases/court-case-alphabetical-index/) to their dedicated pages. This will allow us to delete the "opinions" column from the alphabetical list.

### Related issues

_List any relevant related issue(s)_

- #4204 
- #4210 

### Completion criteria

_Court case alpha page at https://www.fec.gov/legal-resources/court-cases/court-case-alphabetical-index/ lists case names only._

- [x] Copy court decisions and paste onto their dedicated pages. (Pages may need clean up per #4204.) 
- [x] Ensure all links work.
- [ ] Delete "Opinion(s)" column when all decisions have been copied to their dedicated pages.

### Tech steps or considerations (optional)

_List any considerations the tech team should know. Additionally, any specific tech steps can be included here

### Future work

_Other future work that may be needed following this issue's completion._

**Comment by zsmith-fec** [comment]
Stopped at John Hagelin, et al. v. FEC

**Comment by zsmith-fec** [comment]
Stopped at Toledano: FEC v.

**Comment by zsmith-fec** [comment]
Double checking list. Will need second review.

**Comment by zsmith-fec** [comment]
Redoing edits due to Wagtail caching error. Stopped at Doe v. FEC.

**Comment by zsmith-fec** [comment]
Redoing edits due to Wagtail caching error. Fixing footnotes according to new style. Starting from A.

**Comment by zsmith-fec** [comment]
Finished court decision migration and styling A-C. OOO until 12/31. Will pick up with D at that time.

**Comment by DMichaelFEC** [comment]
reviewed and published provided edited cases within A-C on 12/28.

**Comment by zsmith-fec** [comment]
Finished court decision migration and styling D-H. Will pick up with I on 1/4/21.

**Comment by zsmith-fec** [comment]
Finished I through McConnell v. FEC. These cases are ready for review. Will pick up with the rest of M on 1/5/21.

**Comment by zsmith-fec** [comment]
Finished Ms through Robinson v. FEC. These cases are ready for review. Will pick up with FEC v. Rodriguez on 1/6/21.

**Comment by djgarr** [comment]
Reviewed D-R. Everything looks good

**Comment by zsmith-fec** [comment]
All relevant cases updated, reviewed and published. Waiting on litigation to approve deletion of "decisions" column.

**Comment by djgarr** [comment]
Updated H and I cases

**Comment by zsmith-fec** [comment]
Blocked. Waiting on litigation to approve deletion of "decisions" column.

**Comment by zsmith-fec** [comment]
We could not secure approval from litigation to remove the decisions column from the alphabetical list. The ticket will be closed. The issue can be revisited at a later date if necessary. @dorothyyeager @patphongs
